src/ovs/aca_ovs_l2_programmer.cpp

```diff
@@ -454,7 +455,7 @@ int ACA_OVS_L2_Programmer::create_port(const string vpc_id, const string port_na
     if (command_rc != EXIT_SUCCESS)
       overall_rc = command_rc;
   }
-
+/*
```

Comment:
It said 'disable for TC env', means we need to re-enable for master ACA?

Reply:
Yes, it is now enabled.